setup.py

```diff
@@ -69,7 +69,10 @@
     description="clisops - climate simulation operations.",
     license=__license__,
     python_requires=">=3.6.0",
-    install_requires=requirements,
+    install_requires=[
+        requirements,
+        "roocs-utils @ git+https://github.com/roocs/roocs-utils.git",
```

Comment:
I can see this causing problems in the future. Is there any interest in packaging this on PyPI?

Reply:
For some reason, this doesn't fetch the master branch for me. Looking into this.

Reply:
Conda issue. Working for me now! Still, this should be updated to use PyPI.

Reply:
Yes - we plan to put it on PyPi but wanted to get these changes sorted first and will update once its ready if thats ok?

Reply:
Works for me! Just wanted to raise a note here. Managing conda and PyPI and git-installed packages can lead to strange behaviour.